README.md

```diff
@@ -122,11 +122,12 @@ at the [performance](performance/README.md) page.
 
 See the [contributing guide](CONTRIBUTING.md) on how you can help improve `zq`!
 
-[doc-img]: https://godoc.org/github.com/brimsec/zq?status.svg
-[doc]: https://godoc.org/github.com/brimsec/zq
-[ci-img]: https://circleci.com/gh/brimsec/zq.svg?style=svg
-[ci]: https://circleci.com/gh/brimsec/zq
-
 ## Join the Community
 
 Join our [Public Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg) workspace for announcements, Q&A, and to trade tips!
+
+[doc-img]: https://godoc.org/github.com/brimsec/zq?status.svg
+[doc]: https://godoc.org/github.com/brimsec/zq
+[tests-img]: https://github.com/brimsec/zq/workflows/Tests/badge.svg
+[tests]: https://github.com/brimsec/zq/workflows/Tests
```

Comment:
Images look good now.

https://github.com/brimsec/zq/workflows/Tests is 404 for me. I was able to get something reasonable with https://github.com/brimsec/zq/actions?query=workflow%3ATests .

Reply:
I should have checked this. Thanks!